tokio-trace/src/span.rs

```diff
@@ -1,6 +1,50 @@
-//! Spans represent periods of time in the execution of a program.
+//! Spans represent periods of time in which a program was executing in a
+//! particular context.
 //!
-//! # Entering a Span
+//! A span consists of [fields], user-defined key-value pairs of arbitrary data
+//! that describe the context the span represents, and [metadata], a fixed set
```

Comment:
Other docs refer to these just as attributes. Should we use "metadata" everywhere, or "attributes" everywhere?

Reply:
Actually, aren't the two distinct? Is there a difference between `Attributes` and `Metadata`? What is it? Which do you mean here?

Reply:
Is `Metadata` attributes _and_ fields maybe?